### ISSUE TITLE
possible documentation typo

### ISSUE THREAD

**Issue by ochriste** [Pylons/SQLAHelper#2]
hello
Im just wondering if the line
engine = sqlahelper.get_dbengine()
in the documentation shouldnt be
engine = sqlahelper.get_engine()
?

**Comment by rskumar** [comment]
Yes that is typo. But this project is dead for long. No updates.
